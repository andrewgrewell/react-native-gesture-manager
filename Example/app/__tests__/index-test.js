import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import App from '../App';


it('renders the app root correctly', () => {
    const tree = renderer.create(
        <App />
    );
    expect(tree).toMatchSnapshot();
});
