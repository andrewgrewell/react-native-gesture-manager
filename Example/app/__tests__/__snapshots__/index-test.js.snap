exports[`test renders the app root correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "flexDirection": "column",
      "justifyContent": "flex-start",
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
    }
  }>
  <View
    style={
      Object {
        "alignItems": "center",
        "alignSelf": "stretch",
        "backgroundColor": "rgb(44, 217, 222)",
        "flex": 1,
        "justifyContent": "center",
      }
    }>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail">
      Hello World
    </Text>
  </View>
</View>
`;
